### ISSUE TITLE
sending mail from catchall account

### ISSUE THREAD

**Issue by grauschnabel** [mail-in-a-box/mailinabox#1170]
Hi there,

I would like to have a catchall address which is able to send mail for all mailadresses from the domain.  how can I do that?

I think i checked the right option on createing the catchall alias, but I get sender address is not owned by catchall@mydomain.com

Thanks for your help

**Comment by bronson** [comment]
I'm not quite sure what you mean...  You have a catchall user, and you'd like that user to be able to send mail as any user on your system, not just "catchall@mydomain.com"?

If so, that's not currently supported.

If you can explain why you want this, there might be a different way to solve the problem.

**Comment by hachre** [comment]
I think what he meant is that he has a catchall, for example @example.com and he wants the user who receives the catchall mails to be able to write mail as anything@example.com without having to set that up via aliases additionally.

**Comment by bronson** [comment]
Isn't that the same as the second sentence I wrote?  If so, I stand by my answer.  :)

**Comment by hachre** [comment]
It's not quite the same because you saying any user on the system implies other mail accounts on other domains. He might only want that one catchall destination user to be able to send as anything on the catch all domain.

**Comment by hachre** [comment]
I have done some testing and came to the conclusion that the use case (as I've understood it) is supported. 

All I did was:
1. Set up a "catch-all" alias for a domain of mine (@example.com). The radio button was set to "Any mail user listed in the Fowards To box can send mail claiming to be from any address on the alias domain." while doing so.
2. Login into RoundCube with the user that was set up as the recipient of the previous catch-all alias.
3. Set up a new identity within RoundCube that was using a random mail address within the catch-all (test@example.com).

Result was that I was able to send outbound mails as "test@example.com" without any problem or additional hassle.
